packages/ui/scripts/sync-peer-dependencies.js

```diff
@@ -0,0 +1,54 @@
+const process = require('node:process');
+const fs = require('node:path');
+
+const [_x, script, action, file] = process.argv;
+
+if (action !== 'check' && action !== 'fix') {
+    console.error(`Unexpecte action: ${action}`);
+    showHelpAndExit(1);
+}
+
+const {PWD} = process.env;
+
+function showHelpAndExit(exitCode) {
+    console.log('\nUsage:');
+    console.log('\t', process.argv0, fs.relative(PWD, script), 'check', file);
+    showFixHelp(exitCode);
+}
+
+function showFixHelp(exitCode) {
+    console.log('\t', process.argv0, fs.relative(PWD, script), 'fix', file);
+    console.log('');
+    process.exit(exitCode);
+}
+
+const packageJson = require(file.startsWith('/') ? file : PWD + '/' + file);
```

Comment:
I would add a try/catch block here since "require" may not find the file you need.

`try {
  const filePath = file.startsWith('/') ? file : `${PWD}/${file}`;
  const packageJson = require(filePath);
  // .......
} catch (error) {
  console.error(`Error loading file: ${file}`, error);
   // .......
}`

Reply:
It is OK to fail in such case, the script should not continue code-execution.